certificate-policy.md

```diff
@@ -171,16 +117,22 @@ This Certificate Policy for Baseline Requirements for the Issuance and Managemen
 
 
 ### 1.5.1 Organization Administering the Document
-No stipulation.
+The Federal Public Key Infrastructure Policy Authority (FPKIPA) is a group of U.S. Federal Government Agencies and is chartered by the US Federal CIO Council.  The FPKIPA owns this policy and represents the interest of the Federal CIOs.  The FPKIPA is responsible for:  
+* Maintaining this CP, and
+* Approving the CPS for each CA that issues certificates under this policy, and
+* Approving the compliance audit report for each CA issuing certificates under this policy, and
+* Ensuring continued conformance of each CA that issues certificates under this policy with applicable requirements as a condition for allowing continued participation, and
+* Ensuring compliance with CAB Forum Baseline Requirements
+
 
 ### 1.5.2 Contact Person
-Contact information for the CA/Browser Forum is available here:  https://cabforum.org/leadership/.  In this section of a CA's CPS, the CA shall provide a link to a web page or an email address for contacting the person or persons responsible for operation of the CA.
+Contact information for the Federal Public Key Infrastructure Policy Authority is **TODO: Insert Contact Info**. 
 
 ### 1.5.3 Person Determining CPS suitability for the policy
-No stipulation.
+Federal Public Key Infrastructure Policy Authority
 
 ### 1.5.4 CPS approval procedures
-No stipulation.
+Independent Auditors conduct assessments of CPS conformance to the CP requirements.  The results of these audits are submitted and approved by the Federal Public Key Infrastructure Policy Authority.
```

Comment:
I know this is not the way the Fed PKI works but having the compliance auditor do compliance analysis is too late.  Need an approved CPS to stand up operation - audit happens (at best) just prior to commencing operation - at which time, it may be very expensive to fix thing.  DoD has someone actually do a compliance analysis before the PMA approves the CPS - then the auditor checks against that.  Frankly, from what I have seen looking at several fed agency CPS, compliance auditors miss a lot...